lib/notification/adapter/telegram.js

```diff
@@ -15,49 +15,111 @@ const arrayChunks = (inputArray, perChunk) =>
     all[ch] = [].concat(all[ch] || [], one);
     return all;
   }, []);
-function shorten(str, len = 30) {
+
+function shorten(str, len = 45) {
   return str.length > len ? str.substring(0, len) + '...' : str;
 }
-export const send = ({ serviceName, newListings, notificationConfig, jobKey }) => {
+
+function isValidURL(url) {
+  try {
+    new URL(url);
+    return true;
+  } catch (_) {
+    return false;
+  }
+}
+
+const sendTelegramMessage = (token, chatId, message, isPhoto = false, photoUrl = '') => {
+  const url = isPhoto
```

Comment:
1. I think a simple if/else would be easier to read here..
2. The api endpoint /sendPhoto works like /sendMessage, but it can inlcude a photo url?

Reply:
Thanks for the suggestion, @orangecoding; I replaced the ternary operator. As for the API calls, there are some differences, but for the scope of this project, you're absolutely right - they work identically with only photo URL disparity